lib/acts_as_tenant/model_extensions.rb

```diff
@@ -39,6 +43,22 @@ def self.with_tenant(tenant, &block)
     self.current_tenant = old_tenant
   end
 
+  def self.without_tenant(&block)
+    if block.nil?
+      raise ArgumentError, "block required"
+    end
+
+    old_tenant = self.current_tenant
```

Comment:
I think we should capture the old unscoped value as well, since we might nest the without_tenant blocks.